[2024-11-04T19:23:38.910+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bluebikes_data_pipeline.upload_to_gcs_task_ scheduled__2024-08-01T00:00:00+00:00 [queued]>
[2024-11-04T19:23:38.923+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bluebikes_data_pipeline.upload_to_gcs_task_ scheduled__2024-08-01T00:00:00+00:00 [queued]>
[2024-11-04T19:23:38.923+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-11-04T19:23:38.972+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): upload_to_gcs_task_> on 2024-08-01 00:00:00+00:00
[2024-11-04T19:23:38.995+0000] {standard_task_runner.py:57} INFO - Started process 227 to run task
[2024-11-04T19:23:39.012+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'bluebikes_data_pipeline', 'upload_to_gcs_task_', 'scheduled__2024-08-01T00:00:00+00:00', '--job-id', '1816', '--raw', '--subdir', 'DAGS_FOLDER/***_main.py', '--cfg-path', '/tmp/tmpopmvd_vf']
[2024-11-04T19:23:43.821+0000] {standard_task_runner.py:85} INFO - Job 1816: Subtask upload_to_gcs_task_
[2024-11-04T19:23:43.969+0000] {task_command.py:410} INFO - Running <TaskInstance: bluebikes_data_pipeline.upload_to_gcs_task_ scheduled__2024-08-01T00:00:00+00:00 [running]> on host 724f9f0bb0f4
[2024-11-04T19:23:44.103+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Muskan' AIRFLOW_CTX_DAG_ID='bluebikes_data_pipeline' AIRFLOW_CTX_TASK_ID='upload_to_gcs_task_' AIRFLOW_CTX_EXECUTION_DATE='2024-08-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-01T00:00:00+00:00'
[2024-11-04T19:23:44.773+0000] {upload_to_gcp_.py:18} INFO - Uploading file: /opt/***/cleaned_files/preprocessed_202408-bluebikes-tripdata.csv to GCS bucket: trip_data_bucket_testing
[2024-11-04T19:23:52.566+0000] {upload_to_gcp_.py:33} ERROR - Error uploading /opt/***/cleaned_files/preprocessed_202408-bluebikes-tripdata.csv: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /upload/storage/v1/b/trip_data_bucket_testing/o?uploadType=resumable&upload_id=AHmUCY1aw3ooEgMDsrOE4SjiP3ii3Nt3LmWKst5RU-2zxcwf7arQUM9JQfkovfvj_3I2DPOoFJAKEQs14zXfigImb4Fkj0WUAZdRZt3bIXkz-8-M (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_BAD_RECORD_MAC] sslv3 alert bad record mac (_ssl.c:2570)')))
[2024-11-04T19:23:57.848+0000] {upload_to_gcp_.py:33} ERROR - Error uploading /opt/***/cleaned_files/preprocessed_202408-bluebikes-tripdata.csv: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /upload/storage/v1/b/trip_data_bucket_testing/o?uploadType=resumable&upload_id=AHmUCY1DjtD5brAbnRAshckQN0mb2FOqAEbduVQAgubP83Ntxu5Lco8Y-3g5L_h9VjnNB9bkHWUTY0WHxuz0nSTpAAnN2QUF9hzRS1nfEl34AOII (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_BAD_RECORD_MAC] sslv3 alert bad record mac (_ssl.c:2570)')))
[2024-11-04T19:24:06.821+0000] {upload_to_gcp_.py:33} ERROR - Error uploading /opt/***/cleaned_files/preprocessed_202408-bluebikes-tripdata.csv: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /upload/storage/v1/b/trip_data_bucket_testing/o?uploadType=resumable&upload_id=AHmUCY3WBn3C3oEG_u8X-6SbYZo3VYVUI70KQ-4ntYjvCuptj9hDTVSmRkQeHqjQTlRitNduTRIyRgpRAv_jfPE6QIURo1yG7T86d9h_4L2tFEP_bw (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_BAD_RECORD_MAC] sslv3 alert bad record mac (_ssl.c:2570)')))
[2024-11-04T19:24:19.210+0000] {upload_to_gcp_.py:30} INFO - Uploaded /opt/***/cleaned_files/preprocessed_202408-bluebikes-tripdata.csv to gs://trip_data_bucket_testing/Preprocessed_data/preprocessed_202408-bluebikes-tripdata.csv
[2024-11-04T19:24:19.215+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-11-04T19:24:19.260+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=bluebikes_data_pipeline, task_id=upload_to_gcs_task_, execution_date=20240801T000000, start_date=20241104T192338, end_date=20241104T192419
[2024-11-04T19:24:19.316+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-11-04T19:24:19.355+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
